package.json

```diff
@@ -65,7 +65,8 @@
     "inline-style-prefixer": "^3.0.6",
     "normalize-css-color": "^1.0.2",
     "prop-types": "^15.5.10",
-    "react-timer-mixin": "^0.13.3"
+    "react-timer-mixin": "^0.13.3",
+    "@ecliptic/react-art": "0.15.4-fork.1"
```

Comment:
Why the this fork instead of the one owned by FB?

https://www.npmjs.com/package/react-art

Reply:
No thanks. Let's get the official package updated